parseDataset.js

```diff
@@ -30,18 +36,19 @@ const COLUMNS = {
   7: "ingredient_health",
   8: "ingredient_environment",
   9: "quantity_name",
-  10: "quantity_api_name",
-  11: "quantity_id",
-  12: "quantity_default_weight",
-  13: "quantity_default_weight_per_unit",
-  14: "quantity_default_number_of_units",
-  15: "quantity_image_url",
+  10: "quantity_name_1",
```

Comment:
```suggestion
  10: "quantity_name_singular",
```

Same for the line above

Plus running the script to update the config file